client/apollo/client.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
```

Comment:
just need someone to convert this file to `.tsx`. Feel free to raise an issue for that.